package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "wisee-web",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "prettier": "prettier --check .",
+    "prettier-fix": "prettier --write ."
+  },
+  "dependencies": {
+    "class-variance-authority": "^0.7.0",
+    "next": "13.4.19",
+    "postcss": "8.4.28",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "tailwind-merge": "^1.14.0",
+    "typescript": "5.1.6"
```

Comment:
`postcss`, `tailwind-merge`, and `typescript` should go in devDependencies. Check once for cva as well.

Reply:
- Made a PR for this : #5 
- CVA is a package which we need at runtime so not moving to dev dependencies.